### PR TITLE
Added option externalTrafficPolicy

### PR DESCRIPTION
**Pull request by banschikovde**
#### Is this a new chart
No

#### What this PR does / why we need it:

Added option externalTrafficPolicy in k8s service. 
This solution allows to manage traffic flow to rabbitmq as flexibly as possible.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.41.1
+version: 1.41.2
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: Should increase minor version for new features. `1.42.0`

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
/assign
/ok-to-test

**Review by steven-sheehy — COMMENTED** [review]
/assign
/ok-to-test

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (banschikovde) [reply stable/rabbitmq-ha/Chart.yaml]: done

**Comment by Denissa89** [comment]
thanks!

**Pull request merged**